keps/sig-node/5526-pod-level-resource-managers/README.md

```diff
@@ -0,0 +1,1741 @@
+performance-sensitive workloads. All functionality will be controlled by a new
+`PodLevelResourceManagers` feature gate.
+
+## Motivation
+
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users. The
+motivation section can optionally provide links to [experience reports] to
+demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
+
+The introduction of Pod-Level Resources
+([KEP-2837](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2837-pod-level-resource-spec/README.md))
+allows users to define a resource budget for an entire pod. However, the
+Kubelet's key resource management components—Topology, CPU, and Memory
+Managers—are not yet able to use this pod-level specification as the direct
+basis for NUMA alignment decisions. While the `pod` scope can align a pod based
+on the aggregate of its container requests, it cannot leverage the pod-level
+budget directly. This limits performance and flexibility.
+
+For high-performance computing (HPC), AI/ML, and network function virtualization
+workloads, minimizing memory and CPU latency is critical. This is best achieved
+by ensuring all of a pod's resources are allocated from the same NUMA node.
+Furthermore, many of these workloads consist of a primary, performance-sensitive
+container alongside less critical sidecars. The current per-container model
+lacks the flexibility to manage such mixed-criticality pods as a single,
+NUMA-aligned unit with internal resource divisions.
+
+This KEP addresses these gaps by enabling the resource managers to use
+`pod.spec.resources` to NUMA-align the entire pod. It also introduces a
+partitioning model that allows the pod's resources to be divided between
+containers that require exclusive access and those that can share resources, all
+within a single NUMA alignment.
+
+### Goals
+
+<!--
+List the specific goals of the KEP. What is it trying to achieve? How will we
+know that this has succeeded?
+-->
+
+1.  **Pod-level Aware Hint Generation:** The CPU and Memory managers, acting as
+    hint providers for the Topology Manager, will be updated to recognize and
+    use `pod.spec.resources`. When the scope is `pod`, they will generate a
+    single hint for the entire pod's resource budget. When the scope is
+    `container`, they will continue to generate hints on a per-container basis.
+2.  **Implement Hybrid Allocation Logic:** The CPU and Memory managers will
+    implement a hybrid allocation strategy that is determined by the Topology
+    Manager's scope:
+    *   **Pod Scope:** A single NUMA-aligned resource pool will be managed for
+        the entire pod. This pool will then be partitioned, with exclusive
+        slices allocated to `Guaranteed` containers and a shared pool allocated
```

Comment:
Maybe better to use a different name (here and in other places) to differentiate from shared resources used by non-guaranteed pods.